pkg/policy/identity.go

```diff
@@ -95,6 +95,16 @@ func NewIdentityFromModel(base *models.Identity) *Identity {
 	return id
 }
 
+// GetLabelsSHA256 returns the SHA256 of the labels associated with the
+// identity. The SHA is calculated if not already cached.
+func (id *Identity) GetLabelsSHA256() string {
+	if id.LabelsSHA256 == "" {
+		id.LabelsSHA256 = id.Labels.SHA256Sum()
```

Comment:
How can we guarantee the `LabelsSHA256` is updated when in change a label in `id.Labels`?

Reply:
The labels of an identity never change. If the labels change, a new identity is created/assigned.